software/atomic-ai-calendar-assistant

```diff
@@ -0,0 +1,20 @@
+name: "Atomic - an AI calendar assistant"
+website_url: "https://atomiclife.app"
+source_code_url: "https://github.com/rush86999/atomic"
+description: "Atomic, allows flexible timeblocking with priority, flexible meetings and generating new breaks and reminders powered by ChatGPT. It runs on Autopilot and is adjusted as needed. You can teach your time preferences and priority for recurring tasks and use automatted tagging to modify behavior as well."
```

Comment:
it also has the ability to set time preferences, automate break creation, and handshake links for creating meetings with external users from the app [it uses time preference options instead of simple open slots].

Reply:
It's also a Clockwise and Reclaim ai alternative in terms of functionality

Reply:
can we just add an alternative to Clockwise and Reclaim ai? That should be all :) and go with your suggestions from there to make life easier.

Reply:
Change the description to something smaller and concise and we can offer some feedback where needed.

Reply:
Create flexible time blocks with priority & time preferences. Create flexible meetings, auto breaks, and reminders (alternative to motion, Clockwise and reclaim ai).

Reply:
I like this, feel free to commit it